plugins/GSdx/GSHwHack.cpp

```diff
@@ -93,20 +93,17 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT16 && fi.TPSM == PSM_PSMZ16)
 		{
 			// Texture shuffling must work on openGL
-			if (Dx_only)
-				skip = 5; // Sky texture (depth related). On Direct3D the blue sky texture is shown on the whole screen in front of the player if hack is disabled.
+			// Sky texture (depth related). On Direct3D the blue sky texture is shown on the whole screen in front of the player if hack is disabled.
+			// On PAL it also affects the clipped model display of a character (blue or yellow)(does not work properly even on openGL) the effect is broken and shows in front of the player. On NTCS the effect does not work at all.
```

Comment:
A _very_ minor nitpick. ``NTCS`` -> ``NTSC``

You could also probably spilt the second line into two separate lines, ``(Your wish though)``
```
On PAL. ... (explanation)
On NTSC, .. (explanation)
```